README.md

```diff
@@ -162,39 +230,81 @@ interface IDXOptions {
 
 **Returns** `Promise<Definition>`
 
-#### .getEntryId
+#### .getEntry
 
 **Arguments**
 
 1. `definitionId: DocID`
-1. `did?: string`
+1. `did?: string = this.id`
 
-**Returns** `Promise<DocID | null>`
+**Returns** `Promise<Entry | null>`
 
-#### .getEntry
+#### .getEntryReference
 
 **Arguments**
 
 1. `definitionId: DocID`
-1. `did?: string`
+1. `did?: string = this.id`
 
 **Returns** `Promise<unknown | null>`
 
+#### .getEntryTags
+
+**Arguments**
+
+1. `definitionId: DocID`
+1. `did?: string = this.id`
+
+**Returns** `Promise<Array<string>>`
+
 #### .setEntry
 
 **Arguments**
 
+1. `definitionId: DocID`
+1. `entry: Entry`
```

Comment:
So you would no longer set the actual content? An alternative could be:
```ts
function setEntry(def: DocID, content: any, tags?: Array<string>)
```

Reply:
I guess if tags is not set here it would simply not modify the tags?

Reply:
I'd rather avoid changing meaning between method names and inputs/outputs, so IMO `setEntry` should take an `Entry` as input.
Maybe that can be a private method though, so it's more likely developers would use `setEntryReference`? We could rename it to `setContent` maybe?

Reply:
Maybe we just have `setRef(def: DocID, content: any)`?